pepper-apis/studybuilder-cli/studies/lms/emails/germline-consent-addendum-completed-pediatric.html

```diff
@@ -84,29 +84,29 @@
     <td style="padding: 50px 0 0 0;">
       <p style="font-family: 'Source Sans Pro', sans-serif; text-align: left; font-weight: 400; font-size: 16px; line-height: 24px;">Dear -ddp.proxy.firstName-,</p>
 
-      <p style="font-family: 'Source Sans Pro', sans-serif; text-align: left; font-weight: 400; font-size: 16px; line-height: 24px;">Thank you for being a part of the Leiomyosarcoma Project, and for signing the additional consent form which will allow you to learn about your child's non-tumor (also called “normal” or germline DNA).</p>
+      <p style="font-family: 'Source Sans Pro', sans-serif; text-align: left; font-weight: 400; font-size: 16px; line-height: 24px;">Thank you for being a part of the Leiomyosarcoma Project, and for signing the additional consent form which will allow you to learn about your child's non-tumor (also called “normal” or germline DNA) by having genetic counseling and germline genetic testing.</p>
 
-      <p style="font-family: 'Source Sans Pro', sans-serif; text-align: left; font-weight: 400; font-size: 16px; line-height: 24px;">The next step in the process will be to schedule a telehealth appointment with a genetic counselor from Invitae or their telehealth partner, Genome Medical to discuss the testing. You can do this by visiting this website:
+      <p style="font-family: 'Source Sans Pro', sans-serif; text-align: left; font-weight: 400; font-size: 16px; line-height: 24px;">The next step in the process will be to schedule a phone appointment with Genome Medical to discuss the testing. You can do this by visiting this website:
```

Comment:
Should be "You will need to schedule this appointment by visiting this website:" as per source